gene_finder.py

```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
 """
-YOUR HEADER COMMENT HERE
+First project for Olin Software Design Fall 2017
```

Comment:
It might be more helpful to add some short description of what this project is about.